src/applications/disability-benefits/all-claims/local-dev-mock-api/index.js

```diff
@@ -0,0 +1,170 @@
+const moment = require('moment');
```

Comment:
I adapted the responses in here from the `tests/disability-benefits-helpers.js`. They're only used when you run `yarn mock-api --responses src/applications/disability-benefits/all-claims/local-dev-mock-api/index.js`.